app_feup/lib/view/Pages/login_page_view.dart

```diff
@@ -69,7 +69,7 @@ class _LoginPageViewState extends State<LoginPageView> {
 
     return Scaffold(
         backgroundColor: Theme.of(context).brightness == Brightness.light
-            ? Theme.of(context).accentColor
+            ? Theme.of(context).primaryColor
             : Theme.of(context).scaffoldBackgroundColor,
```

Comment:
There is no need to do an if statement
Theme.of(context).primaryColor
will do the job

Reply:
The primary color on the dark theme is white, so we can't make it the background for the login page, indeed, if we want a dark look.

Reply:
You right!